source/MaterialXTest/MaterialXGenShader/GenShaderUtil.cpp

```diff
@@ -764,7 +767,7 @@ void ShaderGeneratorTester::validate(const mx::GenOptions& generateOptions, cons
                     {
                         const std::string elementNameSuffix(bindingContextUsed ? LAYOUT_SUFFIX : mx::EMPTY_STRING);
 
-                        mx::FilePath path = element->getActiveSourceUri();
```

Comment:
@kwokcb would you have a close look at this. It's about the material test that end with `_look`

Reply:
Hi @krohmerNV,
What you have should be correct. The look example has `xincludes`, so the "active" variant is probably grabbing the `URI `of the child included files instead of the the parent working document (which is what you have now).
Thanks for the fix.